solution.cpp

```diff
@@ -1,23 +1,141 @@
+/*---------------------------
+Matthew Low 
+CPSC 121 Sec 11+12
+CWID: 891556854
+---------------------------*/
 /* ----------------------------------------------------------------------------
- * Copyright &copy; 2015 TODO(your name) <TODO(your email)>
- * Released under the [MIT License] (http://opensource.org/licenses/MIT)
- * ------------------------------------------------------------------------- */
+* Copyright & copy; 2015 Matthew Low <mcorelow@csu.fullerton.edu>
+* Released under the [MIT License] (http://opensource.org/licenses/MIT)
+* ---------------------------------------------------------------------------*/
+// This program receives an integer as input and converts it into a Roman numeral
 
-/**
- * TODO: program description
- */
+#include <iostream>
+using std::cin;
+using std::cout;
+using std::endl;
+#include <string>
+using std::string;
 
-// TODO: `#include`s and function prototypes
+void romanNum(int num);
```

Comment:
i think the function was supposed to return a `string` rather than output the number directly.  but that's okay :-)